includes/admin_import.php

```diff
@@ -4,1000 +4,612 @@
-									<a href="https://www.mapbox.com/pricing/" target="_blank">50,000 free map views</a> / month, Google gives
-									<a href="https://cloud.google.com/maps-platform/pricing/" target="_blank">28,500 free views</a>.
-									That's a lot of traffic!
-								</p>
-								<p>To sign up for Mapbox <a href="https://www.mapbox.com/signup/" target="_blank">go here</a>. You will only need
-									a valid email address, no credit card required. Copy your access token and paste it below:</p>
-								<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px;">
-									<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
-									<div class="input">
-										<input type="text" name="tsml_add_mapbox_key" placeholder="Enter Mapbox access token here">
-									</div>
-									<div class="btn">
-										<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
-									</div>
-								</form>
-
-								<p>Alternatively you may use Google. Their interface is slightly more complex, because they offer more
-									services. <a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">Go here</a>
-									to get a key from Google. The process should only take a few minutes, although you will have to enter a
-									credit card.
-									<a href="https://theeventscalendar.com/knowledgebase/setting-up-your-google-maps-api-key/" target="_blank">Here
-										are some instructions</a>.
-								</p>
-
-								<p>Be sure to:<br>
-									<span class="dashicons dashicons-yes"></span> Enable the Google Maps Javascript API<br>
-									<span class="dashicons dashicons-yes"></span> Secure your credentials by adding your website URL to the list
-									of allowed referrers
-								</p>
-
-								<p>Once you're done, paste your new key below.</p>
-
-								<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px;">
-									<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
-									<div class="input">
-										<input type="text" name="tsml_add_google_maps_key" placeholder="Enter Google API key here">
-									</div>
-									<div class="btn">
-										<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
-									</div>
-								</form>
+		//change user interface
+		if (!empty($_POST['tsml_user_interface']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
+			$tsml_user_interface = sanitize_text_field($_POST['tsml_user_interface']);
+			update_option('tsml_user_interface', $tsml_user_interface);
+			if ($tsml_user_interface == 'tsml_ui') {
+				$tsml_ui = "TSML UI";
+			} else {
+				$tsml_ui = "LEGACY UI";
+			}
+			tsml_alert(__('User Interface Display is now set to <strong>' . $tsml_ui . '</strong>', '12-step-meeting-list') );
```

Comment:
Let's consider changing the wording to "Meeting list display is now ..."

Reason: The current wording is kind of geeky. :-)

Reply:
How about "Switch UI is now set to..."?

Reply:
@tech2serve I've gone with the "Switch UI is now set to..." for now. Will change if this is important to you, as I'm not totally against your suggestion.

Reply:
I'd like to see @brianw-area24 or @97eighty weigh in. I note that in no place do we say explicitly that the UI change only affects the public facing meeting list.